datastream-server/src/main/java/com/linkedin/datastream/server/assignment/LoadBasedTaskCountEstimator.java

```diff
@@ -5,17 +5,40 @@
  */
 package com.linkedin.datastream.server.assignment;
 
+import java.util.HashSet;
 import java.util.List;
+import java.util.Map;
+import java.util.Set;
 
-import org.apache.commons.lang.NotImplementedException;
+import org.apache.commons.lang3.Validate;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import com.linkedin.datastream.server.ClusterThroughputInfo;
+import com.linkedin.datastream.server.PartitionThroughputInfo;
 
 
 /**
  * Estimates the minimum number of tasks for a datastream based on per-partition throughput information
  */
 public class LoadBasedTaskCountEstimator {
+  private final static int BYTES_IN_KB_RATE_DEFAULT = 5;
```

Comment:
Make these fields configurable.

Reply:
I don't think there's need, @vmaheshw. I don't think it's something we will want to tune.